### PR TITLE
remove static and add email

### LINKED ISSUE #81: Yield page replacement
https://github.com/fossasia/meilix/pull/85
If this PR works and one can get the link to the mail then we need to remove the yield with the message that the iso will be mail to `$email` in an `estimated time`

### PR DESCRIPTION
**Pull request by xeon-zolt**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
removes Yield output 

#### Changes proposed in this pull request:

- added email html page and css
- edited app.py for removing yield output and add email page

screenshot from 2017-07-19 09-20-31

Fixes #81 #29

### CODE REVIEW

**Comment by meets2tarun** [comment]
Why there isn't a testing link of Heroku app?

**Comment by xeon-zolt** [comment]
strange it should have auto deployed

**Comment by xeon-zolt** [comment]
screenshot from 2017-07-19 09-56-22
the problem of auto deployment we need to enable it for that we need to add credit card details

File: templates/build.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+<body>
+  	<h2>Email will be sent with the downloadable ISO link once it is ready</h2>
+<main>
+	<div class="msg">1</div>
+	<section class="envelope-top"></section>
+	<section class="envelope-left"></section>
+	<section class="envelope-right"></section>
+</main>
+	<a class="tooltip" href="#">Check your email in 10 minutes!</a>
```
Comment (meets2tarun) [inline templates/build.html]: Why did you provide a hyperlink `href='#'` here which neither redirects to the user email inbox?
@xeon-zolt

**Comment by meets2tarun** [comment]
@mariobehling 
if there is an organization account of FOSSASIA where credit card details are given, it auto deployment issue can be resolved and we can transfer the deployment from our account to FOSSASIA account of Heroku.
Link to issue comment: https://github.com/fossasia/meilix-generator/pull/82#issuecomment-316269412
Link to the issue: https://github.com/fossasia/meilix-generator/issues/33

File: templates/build.html (hunk 1 above)
Comment (xeon-zolt) [inline templates/build.html]: i thought of adding feedback form here for user experience will remove in next pull

**Pull request merged**